target/linux/samsung/files-4.14/drivers/mtd/nand/s5pxx.c

```diff
@@ -0,0 +1,279 @@
+/*
```

Comment:
Will you send this driver upstream so that it can be integrated into the mainline Linux kernel?